rewrite-pattern.js

```diff
@@ -221,7 +221,9 @@ const processTerm = (item, regenerateOptions, groups) => {
 			);
 			break;
 		case 'group':
-			groups.lastIndex++;
+			if (item.behavior == 'normal') {
+				groups.lastIndex++;
```

Comment:
This indentation problem is a GitHub rendering bug: https://github.com/mathiasbynens/regexpu-core/blob/776c4882052254c0cf7f483b99b17f1e676d83b5/rewrite-pattern.js#L224-L226